### ISSUE TITLE
Node artifact building is broken.

### ISSUE THREAD

**Issue by jtattermusch** [grpc/grpc#9030]
Seems broken on all platforms.

```
CC(target) Release/obj.target/grpc/src/core/lib/iomgr/resolve_address_uv.o
../src/core/lib/iomgr/resolve_address_uv.c: In function 'blocking_resolve_address_impl':
../src/core/lib/iomgr/resolve_address_uv.c:146:6: error: 'uv_getaddrinfo_t' has no member named 'addrinfo'
   req.addrinfo = NULL;
      ^
../src/core/lib/iomgr/resolve_address_uv.c:160:38: error: 'uv_getaddrinfo_t' has no member named 'addrinfo'
   err = handle_addrinfo_result(s, req.addrinfo, addresses);
                                      ^
../src/core/lib/iomgr/resolve_address_uv.c:165:10: error: 'uv_getaddrinfo_t' has no member named 'addrinfo'
   if (req.addrinfo) {
          ^
../src/core/lib/iomgr/resolve_address_uv.c:166:24: error: 'uv_getaddrinfo_t' has no member named 'addrinfo'
     uv_freeaddrinfo(req.addrinfo);
                        ^
make: *** [Release/obj.target/grpc/src/core/lib/iomgr/resolve_address_uv.o] Error 1
grpc.target.mk:317: recipe for target 'Release/obj.target/grpc/src/core/lib/iomgr/resolve_address_uv.o' failed
make: Leaving directory '/var/local/git/grpc/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
```

https://grpc-testing.appspot.com/view/Artifacts/job/gRPC_build_artifacts/2331/architecture=x86,language=node,platform=linux/console

Same or similar issue occurs on mac and windows as well.

**Comment by murgatroid99** [comment]
I believe this will be fixed by #8895.

**Comment by murgatroid99** [comment]
That PR has been merged, and the Node artifact build works.
